src/wasm/BUILD

```diff
@@ -0,0 +1,13 @@
+load("//third_party/java/arcs/build_defs:build_defs.bzl", "arcs_ts_test")
+
+
+arcs_ts_test(
+    name = "wasm-api-test",
+    src = "tests/wasm-api-tests.ts",
+    deps = [
+        "//src/wasm/cpp/tests:schemas_srcs",
```

Comment:
This seems odd - depending on a `filegroup` containing schema.arcs, rather than the `arcs_cc_schema` build rule for the same file. (I realise this pattern is already being used in cpp/tests/BUILD.)

Do you know if it has to be like this (and if so, why), or can we just use `arcs_cc_schema` as the dependency target?

Reply:
I think we depend on the file itself because the tests reference the file path via `const schemaFile = 'src/wasm/.../schemas.arcs'`

Reply:
After a few experiments, it seems like referencing the `arcs_cc_schema` target doesn't work. I'll keep trying to get it to work, but we can always proceed with the filegroup approach.

Reply:
That's ok, leave it for now. We can revisit later if needed.